src/main/java/org/takes/facets/fork/FkHitRefresh.java

```diff
@@ -234,10 +229,6 @@ private boolean expired() throws IOException {
         /**
          * Directory contents updated?
          * @return TRUE if contents were updated
-         * @todo #802:30min Continue removing nulls from the codebase,
```

Comment:
@Iprogrammerr Please do not remove this puzzle from the codebase

Reply:
@paulodamaso Why? I have removed null from there.

Reply:
@Iprogrammerr Okay, but next time avoid touching in things that are not completely related to your issue, specially `0pdd` puzzles: maybe someone is already assigned to fix the puzzle you are removing and this can cause confusion.